xray/manager.go

```diff
@@ -132,6 +132,36 @@ func (sm *XrayServicesManager) GetScanGraphResults(scanID string, includeVulnera
 	return scanService.GetScanGraphResults(scanID, includeVulnerabilities, includeLicenses)
 }
 
+// BuildScan will send Xray the given build for scan
```

Comment:
// BuildScan will send Xray the given build for scan
-->
// BuildScan scans a published build-info with Xray.